spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/BigQueryRDDFactory.java

```diff
@@ -151,6 +156,60 @@ public RDD<?> createRddFromTable(
         bigQueryReadClientFactory);
   }
 
+  // Moved from BigQueryRDD.scanTable
+  @VisibleForTesting
+  RDD<InternalRow> createRDD(
+      SQLContext sqlContext,
+      Partition[] partitions,
+      ReadSession readSession,
+      Schema bqSchema,
+      String[] columnsInOrder,
+      SparkBigQueryConfig options,
+      BigQueryClientFactory bigQueryClientFactory) {
+    // Unfortunately we need to use reflection here due to a cyclic dependency issue, and the fact
```

Comment:
I understand that the RDD constructor dependencies are different between Scala 2.12 and 2.13. But, how does the dependency between BigQueryRDD and BigQueryPartition matter here?